recipe/meta.yaml

```diff
@@ -1,24 +1,20 @@
 {% set name = "dask" %}
-{% set version = "2023.6.0" %}
+{% set version = "2023.11.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 980741966ef4d14c62dfb146f315f57d5eaaa6bedc52866f99687bc6054c2d4b
```

Comment:
If it's a meta-package, do we even need a source?

Reply:
For the license file ^^'